source/administration/config-settings.rst

```diff
@@ -994,16 +994,31 @@ Note that the gossip port and gossip protocol are used to determine cluster heal
 Enable Experimental Gossip Encryption
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**True**: All communication through the cluster using the gossip protocol will be encrypted.
+**True**: All communication through the cluster uses the gossip protocol will be encrypted.
 
-**False**: All communication using gossip protocol remains unencrypted.
+**False**: All communication uses gossip protocol remains unencrypted.
 
 The encryption uses AES-256 by default, and it is not kept configurable by design. However, you can manually set the ``ClusterEncryptionKey`` row value in the Systems table. A key is a byte array converted to base64. It should be either 16, 24, or 32 bytes to select AES-128, AES-192, or AES-256.
 
 +--------------------------------------------------------------------------------------------------------------------------------+
 | This feature's ``config.json`` setting is ``"EnableExperimentalGossipEncryption": false`` with options ``true`` and ``false``. |
 +--------------------------------------------------------------------------------------------------------------------------------+    
     
+Enable Gossip Compression
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: All communication through the cluster uses gossip compression to maintain compatibility with older servers in a cluster.
```

Comment:
I'd like to emphasize on the fact that it's true by default to maintain compatibility with olders servers.

Reply:
Updated the **True** definition with this feedback.